Makefile.buildtests

```diff
@@ -261,13 +261,13 @@ info-files: QUITE := 0
 info-files:
 	@( \
 		echo "$(abspath $(shell echo "$(MAKEFILE_LIST)"))" | tr ' ' '\n'; \
-		CSRC="$$($(MAKE) USEPKG="" -Bn | grep -o -e "[^ ]\+\.[csS]$$" -e "[^ ]\+\.[csS][ \']")"; \
+		CSRC="$$($(MAKE) USEPKG="" -Bn | grep -o -e "[^ ]\+\.[csS]$$" -e "[^ ]\+\.[csS][ \']" | grep -v -e "-DRIOT_FILE_.*")"; \
```

Comment:
Might be even more robust if we grep for `"^-D"` to remove all precompiler macros ending in .[csS]